packages/cli/src/lib/bundler/optimization.ts

```diff
@@ -24,6 +25,15 @@ export const optimization = (
 
   return {
     minimize: !isDev,
+    ...(options.parallel !== undefined
```

Comment:
Do we not always want to use specifically terser, even when parallelism is disabled? Or maybe that's already the default

Reply:
@freben - Yes, I believe terser is used by default, but in order to control the parallel settings it needs to be added explicitly. There is probably little harm in always including it as `true` by default but I figured this would ensure no odd side effects unless you want to control the parallel setting.

Reply:
I've confirmed terser `~1.4.3` is a dependency of webpack 4. It's a bit odd because `terser-webpack-plugin` is now on `v4.2.3` (which is used by webpack 5). I've pinned the version to the webpack 4 version for now.